js/main.js

```diff
@@ -2,8 +2,8 @@
  * Created by ivanivic on 05/05/16.
  */
 // TIMEZONE
-var timeElement = document.getElementById("time-of-day");
-var timezoneElement = document.getElementById("timezone");
+var timeElement = $(".time-of-day")[0];
+var timezoneElement = $(".timezone")[0];
```

Comment:
ostaviti cu jedan komentar ali vrijedi za sve, ako imas jedinstvene elemente koje zelis selektirati, mozes ostaviti ID.

Kod JS-a je ok ID jer ces osigurati da se na tocnom elementu prikaze, a ovo indeksiranje ti ne osigurava da ce uvijek prvi `.time-of-day` element u DOM-u biti onaj gdje ti zelis prikazati datum.

Kod CSS-a, klase omogućavaju ponovno korištenje CSS pravila.